src/interfaces/filing-interfaces/filing-interfaces.ts

```diff
@@ -103,7 +103,7 @@ export interface RegistrationFilingIF {
       phone: string
     }
     nameRequest: RegistrationNameRequestIF
-    parties: OrgPersonIF[]
+    parties: PartyIF[]
```

Comment:
Here the IF matters because PartyIF names a field "taxId" while OrgPersonIF names it "businessNumber".